fxapom/fxapom.py

```diff
@@ -54,6 +58,29 @@ def create_account(self):
         self.session.verify_email_code(m["headers"]["x-verify-code"])
         return self
 
+    def delete_account(self):
+        try:
+            self.account.clear()
+            self.client.destroy_account(self.email, self.password)
+        except ClientError as err:
+            # 'Unknown Account' error is ok - account already deleted
+            # https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-format
+            if err.errno == 102:
+                return
+            raise
+
+    def login(self):
+        try:
+            session = self.client.login(self.email, self.password)
+            return session
+        except ClientError as err:
+            # 'Unknown Account' error is the only one we care about and will
+            # cause us to throw a custom exception
+            # https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-format
+            if err.errno == 102:
+                raise AccountNotFoundException('FxA Account Not Found')
```

Comment:
Out of curiosity, what does the ClientError message look like? I assume it's not particularly friendly, hence the need for our custom exception?